### PR TITLE
Added SVM Code to Machine Learning

### PR DESCRIPTION
**Pull request by ArchanGhosh**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Using SkLearn's Built-in tools created a notebook that compares the multiple SMV models on the IRIS dataset

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Python/README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@
 * [Logistic Regression](ml/Logistic_Regression.ipynb)
 * [Numeric Outlier Selection](ml/Numerical_outlier_method.ipynb)
 * [Principal Component Analysis](ml/principal_component_analysis.ipynb)
+* [SVM on IRIS Dataset](ml/SVM_ON_IRIS_DATASET.ipynb)
```
Comment (HarshCasper) [inline Python/README.md]: Rename to only `Support Vector Machines`. Datasets really don't matter here!

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Code looks super clean. Good Job 🚀

File: Python/README.md (hunk 1 above)
Comment (ArchanGhosh) [reply Python/README.md]: Okkkk

Comment (ArchanGhosh) [reply Python/README.md]: Changed Filename from SVM_on_IRIS_Dataset to Support Vector  Machine

Changed Readme.md in Python sub-folder for correct  re-direction.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**